cluster.tf

```diff
@@ -63,4 +63,16 @@ resource "aws_instance" "cluster" {
   tags = {
     Name = "${var.environment}-teleport"
   }
-}
\ No newline at end of file
+}
+
+resource "aws_eip" "lb" {
```

Comment:
What's the meaning of `lb` here? I see the EIP is attached to an instance and not a Load Balancer, so just wanted to understand the naming.

Thanks!

Reply:
I renamed aws_eip with cluster